### PR TITLE
[DX-586] Refactor Developer Support Landing Page Content : Add Links

### PR DESCRIPTION
**Pull request by dcs3spp**
[DX-586](https://tyktech.atlassian.net/browse/DX-586) Refactor developer support landing page

[preview](https://deploy-preview-3082--tyk-docs.netlify.app/docs/nightly/frequently-asked-questions/faq/)

[DX-586]: https://tyktech.atlassian.net/browse/DX-586?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/frequently-asked-questions/faq.md [hunk 1]
```diff
@@ -1,13 +1,54 @@
...
 
-Check this section for troubleshooting common issues, in addition to details relating to release notes and upgrading.
+This section provides an overview of the various developer support mechanisms
+offered by Tyk to assist with troubleshooting and frequently asked questions.
 
-* [Release Notes]({{< ref "release-notes" >}})
-* [Upgrading]({{< ref "upgrading-tyk" >}}) information
\ No newline at end of file
+## Support
+
```
Comment (yurisasuke) [inline tyk-docs/content/frequently-asked-questions/faq.md:20]: I wonder if we should add this link (https://support.tyk.io/hc/en-gb) somewhere in this section.

File: tyk-docs/content/frequently-asked-questions/faq.md [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
+- **Announcements**: Use the [announcements](https://community.tyk.io/c/announcements/) channel to learn about release announcements
+- **Meta feedback**: Use the [meta feedback](https://community.tyk.io/c/meta) channel to submit bug reports for Tyk products and give feedback relating to the community forum.
+
+Furthermore, the [Tyk OSS Gateway](https://github.com/TykTechnologies/tyk)
+repository can be used to submit bug reports and feature requests.
+
+### Licensed
+
+Tyk Self Managed and Tyk Cloud license holders can reach out to the online
+[helpdesk](https://support.tyk.io/hc/en-gb) for 24/7 support.
```
Comment (dcs3spp) [inline tyk-docs/content/frequently-asked-questions/faq.md:47]: @keith the link mentioned is in licensed section

File: tyk-docs/content/frequently-asked-questions/faq.md (hunk 1 above)
Comment (dcs3spp) [reply tyk-docs/content/frequently-asked-questions/faq.md:20]: @keith Yep, currently, it is included in the license section. I have added a comment on the line where the link is

File: tyk-docs/content/frequently-asked-questions/faq.md (hunk 2 above)
Comment (yurisasuke) [reply tyk-docs/content/frequently-asked-questions/faq.md:47]: Thank you I have seen it.

**Review by yurisasuke — APPROVED** [review]

**Pull request merged**